Jint/ArrayExt.cs

```diff
@@ -0,0 +1,18 @@
+
+namespace System
+{
+    internal static class ArrayExt
```

Comment:
Can't we make it conditional to NET45 and use the same extension method name as netstandard? `Emtpy<string>()`

Reply:
This is currently not possible. There is no way to define static extension methods.

Reply:
Just for reference: https://github.com/dotnet/roslyn/issues/996 :smile: